ts/models/message.ts

```diff
@@ -1178,16 +1272,27 @@ export class MessageModel extends Backbone.Model<MessageAttributes> {
       }
       const expiresAt = start + delta;
 
-      this.set({ expires_at: expiresAt });
+      this.set({
+        expires_at: expiresAt,
+      });
       const id = this.get('id');
       if (id) {
         await this.commit();
       }
 
-      window?.log?.info('Set message expiration', {
+      window?.log?.info('WIP: Set message expiration', {
```

Comment:
you can .debug those WIP logs if you want to keep them

Reply:
I've cleaned up the WIP logs and changed to .debug where appropriate. I left the WIP for the sync message one as a reminder that we still need to deal with it.